src/promptflow/promptflow/_sdk/_start_trace.py

```diff
@@ -26,8 +31,10 @@ def start_trace():
 
     Note that this function is still under preview, and may change at any time.
     """
-    # TODO: PFS liveness probe
-    pfs_port = get_port_from_config()
+    pfs_port = get_port_from_config(create_if_not_exists=True)
+    _start_pfs_in_background(pfs_port)
```

Comment:
shall we check the pfs port alive before start it?

Reply:
yeah, have checked whether the port can be connected